tests/conftest.py

```diff
@@ -213,8 +213,14 @@ def _gcc_compile(src_file, output_file):
 def aux_bin_paths(test_session_root_path):
     """Build external tools.
 
-    They currently consist of a binary that can properly use the clone()
-    syscall, and a very simple vsock client/server application.
+    They currently consist of:
```

Comment:
nit: The commit message starts as an enum with only one item. I would leave it as a simple sentence.